### PR TITLE
Mail doublon bénéficiaire

### PR DESCRIPTION
**Pull request by NilsRec**
[Ticket](https://trello.com/c/ANNWOIV8/4621-8-mail-doublon-b%C3%A9n%C3%A9ficiaire)

### CODE REVIEW

File: src/Command/DataMigration/MigrateRegionsToTableCommand.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            $this->em->persist($region);
+        }
+        $this->em->flush();
+    }
+
+    private function migrateRegions(OutputInterface $output): void
+    {
+        $regionRepo = $this->em->getRepository(Region::class);
+        $centresRepo = $this->em->getRepository(Centre::class);
+        $centres = $centresRepo->findAllWithRegionAsString();
```
Comment (AntoineAresu) [inline src/Command/DataMigration/MigrateRegionsToTableCommand.php:72]: Not that important, but you don't need intermediate variable

File: src/ServiceV2/Mailer/MailerService.php [hunk 2]
```diff
@@ -30,8 +34,10 @@ public function __construct(
         private readonly RouterInterface $router,
         private readonly LoggerInterface $logger,
         private readonly TranslatorInterface $translator,
+        private readonly Security $security,
         private readonly string $mailerSender,
         private readonly array $adminMails,
```
Comment (AntoineAresu) [inline src/ServiceV2/Mailer/MailerService.php:39]: We don't need to inject adminMails anymore

File: .env [hunk 3]
```diff
@@ -52,6 +52,7 @@ BUCKET_SECRET=minioadmin
 
 ###> symfony/mailer-bundle ###
 ADMIN_MAILS='["antoine.aresu@reconnect.fr", "celine.gacon@reconnect.fr", "nils.lapotre@reconnect.fr", "thibaut.cheymol@reconnect.fr"]'
+DUPLICATE_DEFAULT_RECIPIENT='respo@antennes.fr'
```
Comment (AntoineAresu) [inline .env:55]: REGIONAL_MANAGER_MAIL could be clearer ?

File: src/Command/DataMigration/MigrateRegionsToTableCommand.php (hunk 1 above)
Comment (NilsRec) [reply src/Command/DataMigration/MigrateRegionsToTableCommand.php:72]: $centres ? I actually use it to set the progress bar

File: src/ServiceV2/Mailer/MailerService.php (hunk 2 above)
Comment (NilsRec) [reply src/ServiceV2/Mailer/MailerService.php:39]: oh nice !

Comment (NilsRec) [reply src/ServiceV2/Mailer/MailerService.php:39]: Well, we need it for personal data request emails.

File: src/ServiceV2/Mailer/Email/DuplicatedUsernameEmail.php [hunk 4]
```diff
@@ -2,20 +2,28 @@
 
 namespace App\ServiceV2\Mailer\Email;
 
+use App\Entity\Centre;
+use App\Entity\Region;
 use App\Entity\User;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
 use Symfony\Component\Mime\Email;
 
 readonly class DuplicatedUsernameEmail
```
Comment (AntoineAresu) [inline src/ServiceV2/Mailer/Email/DuplicatedUsernameEmail.php:11]: Why didn't you extends `LocalizedTemplatedEmail` ?

File: .env (hunk 3 above)
Comment (NilsRec) [reply .env:55]: Oh no i actually forgot to change it. It's support@reconnect.fr.

File: src/ServiceV2/Mailer/Email/DuplicatedUsernameEmail.php (hunk 4 above)
Comment (NilsRec) [reply src/ServiceV2/Mailer/Email/DuplicatedUsernameEmail.php:11]: I don't think we ever gonna translate it ?

File: src/ServiceV2/Mailer/MailerService.php (hunk 2 above)
Comment (AntoineAresu) [reply src/ServiceV2/Mailer/MailerService.php:39]: mybad

File: src/ServiceV2/Mailer/Email/DuplicatedUsernameEmail.php (hunk 4 above)
Comment (AntoineAresu) [reply src/ServiceV2/Mailer/Email/DuplicatedUsernameEmail.php:11]: Translation is not mandatory, it could be helpfull to lighten this email class

**Review by tcheymol — APPROVED** [review]

**Pull request merged**